poem/src/server.rs

```diff
@@ -270,6 +290,19 @@ where
     }
 }
 
+trait Http2ConfigCustomize: Send + Sync {
+    fn customize(&self, builder: &mut Http2Builder<'_, hyper_util::rt::TokioExecutor>);
+}
```

Comment:
It’s for simplicity, and it’s also possible to allow passing not only closures, but custom implementations of the customizer if needed